github-dark.css

```diff
@@ -4072,9 +4079,26 @@
   .zhc-btn--primary-bordered:hover, .zhc-btn--secondary-bordered:hover,
   .zhc-btn--secondary:hover, .zhc-issue-card__actions__btn:hover,
   .zhc-btn:hover .zhc-btn--text-secondary:hover, .zhc-filter-expand-button:hover,
-  .zhc-chart-legend__labels--notselected {
+  .zhc-chart-legend__labels--notselected, .zhc-pipeline-header,
+  .zhc-pipeline ::-webkit-scrollbar-track, ::-webkit-scrollbar-corner,
+  .zhc-pipeline-body { {
```

Comment:
Please remove the extra curly bracket here. Thanks!

Reply:
Sorry about that!